server/package.json

```
@@ -0,0 +1,26 @@
+{
+    "name": "lsp-prisma-server",
+    "description": "Implementation of a language server in node.",
+    "version": "0.0.30",
```

```suggestion
    "version": "0.0.31",
```
for consistency - the update script will have to keep this updated longer term

---

I changed that in the client folder as well